<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QWidget" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton[active=&quot;true&quot;] {
  color: red;
  font-weight: bold;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_3">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserWidget</receiver>
   <slot>emitChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>emitChanged()</slot>
 </slots>
</ui>
